### PR TITLE
Feature/request monitoring, priority e land-use

### PR DESCRIPTION
**Pull request by GabrielMarianosf**
Melhorias nas requisições (monitoring, priority e land-use) e validação dos inputs do land-use

### CODE REVIEW

File: components/land-use/LandUseFilter.vue [hunk 1]
```diff
@@ -223,8 +234,24 @@ export default {
         },
 
         search() {
-            this.setFilters(this.filters)
-            this.$emit('onSearch')
+            if (this.filters.cr.length > 0 && this.filters.year.length === 0) {
```
Comment (dagnaldo) [inline components/land-use/LandUseFilter.vue:237]: Improve its if and elses, please...

File: layouts/default.vue [hunk 2]
```diff
@@ -71,22 +71,24 @@
...
-        <div v-if="$store.state.priority.visualizationStage == 'stage4'">
+        <div
+            v-if="$store.state.priority.visualizationStage == 'tableMonitoring'"
+        >
             <v-main class="pa-0">
                 <MonitoringTable />
             </v-main>
         </div>
-        <div v-if="$store.state.priority.visualizationStage == 'stage5'">
+        <div v-if="$store.state.priority.visualizationStage == 'tableLand'">
```
Comment (dagnaldo) [inline layouts/default.vue:91]: Rename it to tableLandUse, following the pattern

File: pages/index/monitoring.vue [hunk 3]
```diff
@@ -106,14 +111,25 @@ export default {
...
     methods: {
         search() {
-             this.getFeatures()
-             this.getDataTableMonitoring()
-            
+            if (this.visualizationStage == 'map') this.getFeatures()
+            if (this.visualizationStage == 'tableMonitoring')
+                this.getDataTableMonitoring()
+        },
+        verifyFeatures() {
```
Comment (dagnaldo) [inline pages/index/monitoring.vue:128]: Rename it to checkFeatures...

File: pages/index/monitoring.vue [hunk 4]
```diff
@@ -106,14 +111,25 @@ export default {
...
+            if (this.tableMonitoring.length) this.getFeatures()
+        },
+        verifyTable() {
```
Comment (dagnaldo) [inline pages/index/monitoring.vue:131]: Rename it to checkTable...

File: pages/index/priority.vue [hunk 5]
```diff
@@ -151,8 +152,14 @@ export default {
 
     methods: {
         search() {
-            if (this.visualizationStage == 'stage1') this.getFeatures()
-            if (this.visualizationStage == 'stage3') this.getDataTable()
+            if (this.visualizationStage == 'map') this.getFeatures()
+            if (this.visualizationStage == 'tablePriority') this.getDataTable()
+        },
+        verifyFeatures() {
```
Comment (dagnaldo) [inline pages/index/priority.vue:158]: Rename it to checkFeatures...

File: pages/index/priority.vue [hunk 6]
```diff
@@ -151,8 +152,14 @@ export default {
...
+            if (this.table.length) this.getFeatures()
+        },
+        verifyTable() {
```
Comment (dagnaldo) [inline pages/index/priority.vue:161]: Rename it to checkTable...

**Review by dagnaldo — CHANGES_REQUESTED** [review]
Please, take a look.

File: pages/index/priority.vue (hunk 6 above)
Comment (GabrielMarianosf) [reply pages/index/priority.vue]: renamed

File: pages/index/priority.vue (hunk 5 above)
Comment (GabrielMarianosf) [reply pages/index/priority.vue]: renamed

File: layouts/default.vue (hunk 2 above)
Comment (GabrielMarianosf) [reply layouts/default.vue]: renamed

File: components/land-use/LandUseFilter.vue (hunk 1 above)
Comment (GabrielMarianosf) [reply components/land-use/LandUseFilter.vue]: improved code

**Review by dagnaldo — APPROVED** [review]

**Pull request merged**